src/test/kotlin/uk/gov/dluhc/printapi/config/IntegrationTest.kt

```diff
@@ -97,9 +99,6 @@ internal abstract class IntegrationTest {
     @Autowired
     protected lateinit var processPrintResponseFileMessageQueue: MessageQueue<ProcessPrintResponseFileMessage>
 
-    @Autowired
```

Comment:
Might need this back in (to prevent a merge conflict). I know it wasnt used, but I put it there ready to be used by @fayimora and @abdullah-sanver-valtech in their current tickets; and I think they'll be wanting it there ¯\_(ツ)_/¯